### PR TITLE
Translate: Typechecking With PropTypes

### PR DESCRIPTION
**Pull request by jnosornov**
Relates to issue #4.

Have some doubts about some therminology translation.

- typecheking
- validator

Check out the PR and let me know.

### CODE REVIEW

File: content/docs/typechecking-with-proptypes.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 permalink: docs/typechecking-with-proptypes.html
 redirect_from:
   - "docs/react-api.html#typechecking-with-proptypes"
 ---
 
-> Note:
+> Nota:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` se movió a un paquete diferente desde React v15.5. Por favor use [en su lugar la libreria `prop-types`](https://www.npmjs.com/package/prop-types).
```
Comment (RamirezAlex) [inline content/docs/typechecking-with-proptypes.md]: Hi @jnosornov 👍 please chance this sentence to: 
Por favor usa [en su lugar la biblioteca `prop-types`](https://www.npmjs.com/package/prop-types).
:)

File: content/docs/typechecking-with-proptypes.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+>Nosotros ofrecemos [un script de codemod](/blog/2017/04/07/react-v15.5.0.html#migrating-from-reactproptypes) para automatizar la conversión.
 
-As your app grows, you can catch a lot of bugs with typechecking. For some applications, you can use JavaScript extensions like [Flow](https://flow.org/) or [TypeScript](https://www.typescriptlang.org/) to typecheck your whole application. But even if you don't use those, React has some built-in typechecking abilities. To run typechecking on the props for a component, you can assign the special `propTypes` property:
+A medida que tú aplicación crece, puedes capturar una gran cantidad de errores con **verificación de tipos**. Para algunas aplicaciones, puedes usar extensiones de Javascript como [Flow](https://flow.org/) o [TypeScript](https://www.typescriptlang.org/) para verificar los tipos en tú aplicación. Pero incluso si no usas alguno de ellos, React tiene algunas habilidades de verificación de tipos incorporadas. Para usar verificación de tipos en las **props** de un component, puedes asignar la propiedad especial `PropTypes`:
```
Comment (RamirezAlex) [inline content/docs/typechecking-with-proptypes.md]: `A medida que tu` has no accent.

File: content/docs/typechecking-with-proptypes.md [hunk 3]
```diff
@@ -50,50 +50,53 @@ MyComponent.propTypes = {
   optionalString: PropTypes.string,
   optionalSymbol: PropTypes.symbol,
 
+  /* doubt */
+  // Cualquier cosa que sea interpretada: numeros, cadenas, elementos o un array
```
Comment (RamirezAlex) [inline content/docs/typechecking-with-proptypes.md]: `números`

File: content/docs/typechecking-with-proptypes.md [hunk 4]
```diff
@@ -50,50 +50,53 @@ MyComponent.propTypes = {
...
 
-  // You can also declare that a prop is an instance of a class. This uses
-  // JS's instanceof operator.
+  // Además puedes declarar que una **prop** es una instancia de una clase. Este usa
+  // el operador instanceof de Js.
   optionalMessage: PropTypes.instanceOf(Message),
 
-  // You can ensure that your prop is limited to specific values by treating
-  // it as an enum.
+  // Puedes asegurar que una **prop** esta limitada a valores especificos si se
```
Comment (RamirezAlex) [inline content/docs/typechecking-with-proptypes.md]: `específicos`

File: content/docs/typechecking-with-proptypes.md [hunk 5]
```diff
@@ -50,50 +50,53 @@ MyComponent.propTypes = {
...
 
-  // A value of any data type
+  // Un valor de cualquier tipo
   requiredAny: PropTypes.any.isRequired,
 
-  // You can also specify a custom validator. It should return an Error
-  // object if the validation fails. Don't `console.warn` or throw, as this
-  // won't work inside `oneOfType`.
+  // También puedes suministrar un validador personalizado. Debe retornar un objeto Error
+  // si la validación falla. No use `console.warn` o throwm, porque no va a funcionar en
```
Comment (RamirezAlex) [inline content/docs/typechecking-with-proptypes.md]: There is a little typo in throw, please remove the m. :)

File: content/docs/typechecking-with-proptypes.md (hunk 1 above)
Comment (jnosornov) [reply content/docs/typechecking-with-proptypes.md]: alright I will modify it

**Comment by RamirezAlex** [comment]
Thank you @jnosornov! Great job! LGTM 👍

File: content/docs/typechecking-with-proptypes.md [hunk 6]
```diff
@@ -50,50 +50,50 @@ MyComponent.propTypes = {
...
 
-  // A value of any data type
+  // Un valor de cualquier tipo
   requiredAny: PropTypes.any.isRequired,
 
-  // You can also specify a custom validator. It should return an Error
-  // object if the validation fails. Don't `console.warn` or throw, as this
-  // won't work inside `oneOfType`.
+  // También puedes suministrar un validador personalizado. Debe retornar un objeto Error
+  // si la validación falla. No use `console.warn` o throw, porque no va a funcionar en
```
Comment (alejandronanez) [inline content/docs/typechecking-with-proptypes.md]:
```suggestion
  // si la validación falla. No uses `console.warn` o throw, porque no va a funcionar en
```

File: content/docs/typechecking-with-proptypes.md [hunk 7]
```diff
@@ -103,11 +103,10 @@ MyComponent.propTypes = {
     }
   },
 
-  // You can also supply a custom validator to `arrayOf` and `objectOf`.
-  // It should return an Error object if the validation fails. The validator
-  // will be called for each key in the array or object. The first two
-  // arguments of the validator are the array or object itself, and the
-  // current item's key.
+  // También puede suministrar un validador personalizado a `arrayOf` y `objectOf`.
```
Comment (alejandronanez) [inline content/docs/typechecking-with-proptypes.md]:
```suggestion
  // También puedes suministrar un validador personalizado a `arrayOf` y `objectOf`.
```

File: content/docs/typechecking-with-proptypes.md [hunk 8]
```diff
@@ -156,19 +155,19 @@ class Greeting extends React.Component {
...
-// Renders "Hello, Stranger":
+// Renderiza "Hello, Stranger":
 ReactDOM.render(
   <Greeting />,
   document.getElementById('example')
 );
 ```
 
-If you are using a Babel transform like [transform-class-properties](https://babeljs.io/docs/plugins/transform-class-properties/) , you can also declare `defaultProps` as static property within a React component class. This syntax has not yet been finalized though and will require a compilation step to work within a browser. For more information, see the [class fields proposal](https://github.com/tc39/proposal-class-fields).
+Si estas usando un transform de Babel como [transform-class-properties](https://babeljs.io/docs/plugins/transform-class-properties/), puedes declarar `defaultProps` como una propiedad estática al interior de un componente clase de React. Esta sintaxis no se ha terminado y require un paso de compilación para funcionar en el navegador. Para mas información, vea [class field proposal](https://github.com/tc39/proposal-class-fields).
```
Comment (alejandronanez) [inline content/docs/typechecking-with-proptypes.md]:
```suggestion
Si estas usando un transform de Babel como [transform-class-properties](https://babeljs.io/docs/plugins/transform-class-properties/), puedes declarar `defaultProps` como una propiedad estática al interior de un componente clase de React. Esta sintaxis no se ha terminado y require un paso de compilación para funcionar en el navegador. Para mas información, puedes ver [class field proposal](https://github.com/tc39/proposal-class-fields).
```

Comment (RamirezAlex) [reply content/docs/typechecking-with-proptypes.md]: Si estás usando.... has an accent ;-)

**Comment by jnosornov** [comment]
thanks guys for the review @RamirezAlex and @alejandronanez. Now move on! 💯

**Pull request merged**